index.bs

```diff
@@ -7,36 +7,86 @@ Level: none
 URL: https://sockets-api.proposal.wintercg.org/
 Repository: https://github.com/wintercg/proposal-sockets-api
 Editor: Dominik Picheta, Cloudflare https://cloudflare.com/, dominik@cloudflare.com
+Editor: Ethan Arrowood, Vercel https://vercel.com/, ethan.arrowood@vercel.com
 Abstract: Sockets API for Non-Browser EcmaScript-based runtimes.
 Markup Shorthands: markdown yes
+Markup Shorthands: idl yes
 </pre>
+
 <pre class=link-defaults>
 spec:url; type:interface; text:URL
 spec:html; type:attribute; for:Window; text:navigator
 spec:html; type:method; for:Connect; text:connect
 </pre>
 
-Introduction {#intro}
-=====================
+<h2 id="intro">Introduction</h2>
+
+<div class="non-normative">
 
-*This section is non-normative.*
+<em>This section is non-normative.</em>
 
 This document defines an API for establishing TCP connections in Non-Browser JavaScript runtime
-environments. Existing standard APIs are reused as much as possible, for example ReadableStream
-and WritableStream are used for reading and writing from a socket. Some options are inspired
+environments. Existing standard APIs are reused as much as possible, for example {{ReadableStream}}
+and {{WritableStream}} are used for reading and writing from a [=socket=]. Some options are inspired
 by the existing Node.js `net.Socket` API.
 
-The API described here is already implemented by Cloudflare Workers.
-Workers-specific documentation is available in
-https://developers.cloudflare.com/workers/runtime-apis/tcp-sockets/. This spec shares many
-similarities with that document.
+</div>
+
+<h2 id="concepts">Concepts</h2>
```

Comment:
I am not sure we need this section. I like how it is in the Streams spec because it lays out the higher level concepts before diving into all the individual (and complex) classes. But this spec is simpler, so I think this is redundant. Should I remove this section and move the explanations into the various other relevant sections?

Reply:
Unless other folks find value in this section, I'm going to remove it. I just tried it out locally and I personally like it more that way. It simplifies the spec and declutters some of the linking nonesense.

Reply:
No strong feelings either way here.